Doc/tutorial/introduction.rst

```diff
@@ -145,7 +145,7 @@ Python can manipulate text (represented by type :class:`str`, so-called
 "strings") as well as numbers.  This includes characters "``!``", words
 "``rabbit``", names "``Paris``", sentences "``Got your back.``", etc.
 "``Yay! :)``". They can be enclosed in single quotes (``'...'``) or double
-quotes (``"..."``) with the same result [#]_.
+quotes (``"..."``) with the same result [#]_.::
```

Comment:
This will appear as `.:`. The rules are:

> * If [`::`] occurs as a paragraph of its own, that paragraph is completely left out of the document.
> * If it is preceded by whitespace, the marker is removed.
> * If it is preceded by non-whitespace, the marker is replaced by a single colon.

This preserves the period:

```suggestion
quotes (``"..."``) with the same result [#]_. ::
```

This replaces it with a colon:

```suggestion
quotes (``"..."``) with the same result [#]_::
```